extensions/amp-consent/0.1/consent-state-manager.js

```diff
@@ -276,11 +301,11 @@ export class ConsentInstance {
   update(state, consentString, opt_systemUpdate) {
     const localState =
       this.localConsentInfo_ && this.localConsentInfo_['consentState'];
-    const localConsentStr =
-      this.localConsentInfo_ && this.localConsentInfo_['consentString'];
     const calculatedState = recalculateConsentStateValue(state, localState);
 
     if (state === CONSENT_ITEM_STATE.DISMISSED) {
+      const localConsentStr =
```

Comment:
?

Reply:
Just moving it closer to where it's used. Should I undo?

Reply:
ohh, nvm.